content/src/editor.less

```diff
@@ -805,19 +818,34 @@ body#pencildoc #diigolet-csm {
 }
 
 .directory .item {
+  text-align: center;
   display: inline-block;
   float: left;
   clear: left;
   text-decoration: none;
-  padding: 0 28px 0 12px;
-  min-height: 28px;
   color: #222;
-  cursor: pointer
+  cursor: pointer;
+  div {
+    width: 200px;
```

Comment:
Good question... With padding it's actually 96x96... But I think storing
128 might be useful later for other ui - showcasing popular projects etc.
On Jun 17, 2015 9:26 AM, "Liu Xinan" <notifications@github.com> wrote:

> In content/src/editor.less
> <https://github.com/PencilCode/pencilcode/pull/133#discussion_r32625356>:
>
> >    color: #222;
> > -  cursor: pointer
> > +  cursor: pointer;
> > +  div {
> > +    width: 200px;
>
> If we are using a 100x100 square for displaying thumbnails, do we still
> want to save the thumbnail in the size of 128x128?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PencilCode/pencilcode/pull/133/files#r32625356>.
>